mailing_list/templates/mailing_list/admin_index.html

```diff
@@ -64,108 +64,91 @@
           </p>
         </div>
 
-        <div class="col-md-12">
-            <!--<hr width="98%">-->
-            <!-- show combined mailing list or single primary list -->
-            <!-- if there is a combined mailing list show it -->
-            <div ng-if="ml.isCourseList(list)" ng-repeat="list in ml.primarySectionLists">
-              {% include "mailing_list/_mailing_list_details.html" with scope="class" %}
-                <div class="col-md-12">
-                    <hr width="98%">
-                </div>
-            </div>
+    <div class="col-md-12">
 
-            <!-- otherwise show the primary section -->
-            <div ng-if="ml.primarySectionLists.length == 1" ng-repeat="list in ml.primarySectionLists">
-              {% include "mailing_list/_mailing_list_details.html" with scope="class" %}
+      <!--<hr width="98%">-->
+      <!-- show combined mailing list or single primary list -->
+      <!-- if there is a combined mailing list show it -->
+      <div ng-repeat="list in ml.courseList">
+        {% include "mailing_list/_mailing_list_details.html" with scope="class" %}
+        <div class="col-md-12">
+          <hr width="98%">
+        </div>
+      </div>
+
+      <div class="btn-group col-md-12">
+        <div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
+          <div class="panel panel-default" ng-show="ml.hasNonEnrollmentSections()">
+            <div class="panel-heading" role="tab" id="headingOne">
+              <h4 class="panel-title">
+                <a class="collapsed"
+                 role="button"
+                 data-toggle="collapse"
+                 data-parent="#accordion"
+                 href="#collapseTwo"
+                 aria-expanded="false" aria-controls="collapseTwo">
+                  <ng-pluralize count="ml.nonEnrollmentSectionLists.length"
+                        when="{'0': 'No Sections for ', 'one': 'Section Mailing List for ', 'other': 'Sections Mailing Lists for '}">
+                  </ng-pluralize>{{course_code}}
+                </a>
+              </h4>
             </div>
-            <!-- end show combined mailing list or single primary list -->
 
-            <div ng-show="!ml.hasPrimarySections()" class="col-md-12">
-              <p>
-                  There was an error retrieving the class mailing list for this course.
-                  Please contact your local academic support staff for assistance.
-              </p>
```

Comment:
Looks like there was a placeholder for displaying this error message on  not finding primary sections and I see that the hasPrimarySections() method is not there anymore. Do  we not need this check for the method that replaced it?(hasEnrollmentSections?). Maybe not, but am wondering if it was for a specific use case.